### ISSUE TITLE
Offhand proccing Mainhand Weapon Imbue

### ISSUE THREAD

**Issue by Involve** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3584]
***Current Situation:***
Currently, a Shaman's Offhand is capable of proccing the Weapon Imbue which was applied to the mainhand.
***Desired Situation:***
The Offhand should only be able to proc the Weapon Imbue which was actually applied to the Offhand Weapon.

***Reproduction Guide:***
In order to reproduce the following issue you will need two clients

Client A:
Create any Class
.gm off
.gm visible on
.levelup 69
.t test 
.aura 1908

Client B
Create a Shaman
.gm off
.gm visible on
.levelup 69
.t test
.learn 196
.learn 30798
.add 34996
.add 34995
.learn 25505
.learn 25500

Equip both Axes on the Shaman and apply Windfury Weapon to the Mainhand and Frostbrand Weapon to the Offhand. Start autoattacking Client A with both weapons enchanted and pay attention to the combat log. You will notice that you are capable of proccing Windfury, which is enchanted on the Mainhand, by hitting your target with an Offhand Swing.

![1](https://cloud.githubusercontent.com/assets/6666211/7447806/0adbeb4c-f209-11e4-9243-101b31b47782.jpg)

**Comment by Doppelmoral** [comment]
Nice finding.

As a sidenote btw: There's more off with Windfury. An Enh can chainprocc Wf when using a 2H-Weapon with a (flurried) speed of < 3,0

Means: Hit -> Wf procc
2,77 sec (3,60 -30%) -> Hit -> Wf procc

Might be worth looking into it as well, cba making new report about that and repro :x

**Comment by Nifire** [comment]
If you stat looking at windfury, you should look into why double windfury imbue don't work on your server.

**Comment by Involve** [comment]
You should maybe explain what exactly does not "work", because apart from the wrong scaling of the Offhand and the bug mentioned here I am not aware of anything else.

**Comment by teddys** [comment]
I think thats what he meant (the dmg scaling)

**Comment by Nifire** [comment]
All i can really tell you is it does not work, I tested it a lot why it does not work I don't know. What i do know is double windfury with 2x slow weapons is suppose to beat all other combination in dps. Current state is it does not. Best dps is fast oh and FT. 

What could also be an issue I guess is the bug described here. Hf oh can trig MH WF I assuse it also work the other way around.

**Comment by Involve** [comment]
You realise that this might not necessarily be related to Windfury being too weak, but possibly Flametongue simply being better than its supposed to be?

Either way - I'm aware that the OH WF Scaling is off. What I'm not sure about is whether the FT Scaling is fine or if its too strong for whatever reason.

**Comment by Doppelmoral** [comment]
Flame Tongue "suffers" from the same problem that all spellpower scalibgs have. They do not consider penalties but use the 100% formula.

Best examples are Crusader enchant or any low level effects that should have a decreased effect on higher characters.

FT should do less damage on faster weapons and more on slower weapons to sustain a static DpS (people say it works, haven't tested it myself). This mechanic us not functional in most cases (Crusader having 10% rate and 100 str for 70 while it should gradually diminish to 2% and 60 str on max. level or trinkets that won't fail increasingly on targets > lvl 35.)

At any rate this report is about OP's issue, for the sake of simplicity let's focus on that before bothering with FT at all.

**Comment by Blaqwar** [comment]
Yes when it comes to small changes let's nerf before we buff cause that keeps players happy, after all, they are the lifeblood of the server.

The decreased scaling as you describe it in general does work, I can confirm that because I have an undergeared 70 alt running around with a Tidal Charm.

**Comment by Zerixx** [comment]
This was never applied to PTR/Dev branch. Coming in my next commits.
